### DISCUSSION TITLE
Generic ARM image request

### DISCUSSION THREAD

**Discussion by Arthur2500** [MichaIng/DietPi#5978] [Image Request] [answered]
#### Formal device information
- Device name | Generic ARM(v6-v9)
- Official product URL | N/A
- GitHub resource URL (if available) | 
- Image download URL | https://cdimage.debian.org/debian-cd/current/arm64/iso-cd/debian-11.5.0-arm64-netinst.iso

#### Is the SBC officially supported by the Debian installer?
- yes

#### If not, is a reliable 3rd party Debian image available for this SBC?
- ...

#### If not, are there install instructions for Debian available?
- yes

I am requesting a general-use ARM image, similar to the "Generic x86" image provided on the website. This would make creating virtual machines on ARM-Systems easier.

**Comment by makinikm** [comment]
Hi, 
I know this is a bit of necroposting here, but this was already tagged and its directly related...
I've started using the `testing` VM image for ARMv8 with UEFI - [DietPi_VM-UEFI-ARMv8-Trixie.img.xz](https://dietpi.com/downloads/images/testing/) ...And it worked out to be the only painlessly "deployable" solution for VM base images, so far. Some other images simply required too much work before booting, some just didn't boot or were horribly unoptimized and slow if did, even ones from The Upstream project. Simple qcowing and extending the rootfs was enough with that one to boot it in qemu on Opi5Max DP 1.5.2 7.0.11-edge host, the guest:
```
Startup finished in 1.771s (kernel) + 2.646s (userspace) = 4.417s 
graphical.target reached after 2.644s in userspace.
``` 
This is nearly perfect for ARM VMs on Pi, and it turned out to be "ci-friendly" even for home lab environments!
Emm... could this image become a maintained thing? 🙏🏼

**Comment by MichaIng** [comment, marked as answer]
We have ARMv8 VM images now, raw, QCOW2, and various appliances. And we have network-capable container images. This should cover this request. For physical boards, a generic image is simply impossible, due to the individual bootloader and device tree needs.
